src/Homework_7/Homework_7/task_7_2.py

```diff
@@ -0,0 +1,143 @@
+def duim_v_sant(duim):
+    sant = duim * 2.54
+    print(f"В {duim} дюймах {sant} сантимантв")
+    return sant, duim
```

Comment:
Пусть функция возвращает только результат операции 
```suggestion
    return duim * 2.54
```